DatadogWebViewTracking/Sources/WebViewTracking.swift

```diff
@@ -25,28 +25,61 @@ import WebKit
 /// - Support users that have difficulty loading web pages on mobile devices
 public enum WebViewTracking {
 #if !os(tvOS)
-    /// Enables SDK to correlate Datadog RUM events and Logs from the WebView with native RUM session.
+    /// The Session Replay Configuration to capture records coming from the web-view.
```

Comment:
To unify the language with other API comments:
```suggestion
    /// The Session Replay configuration to capture records coming from the web view.
```